Ginger/Ginger/Environments/AppDataBasesPage.xaml.cs

```diff
@@ -236,15 +227,34 @@ private void AddNewDB(object sender, RoutedEventArgs e)
         {
             Database db = new Database();
             db.Name = "New";
+            db.PropertyChanged += db_PropertyChanged;
             grdAppDbs.DataSourceList.Add(db);
         }
 
         private void SetGridData()
         {
+            foreach (Database db in AppOwner.Dbs)
+            {
+                db.PropertyChanged -= db_PropertyChanged;
+                db.PropertyChanged += db_PropertyChanged;
+            }
             grdAppDbs.DataSourceList = AppOwner.Dbs;
         }
         #endregion Functions
 
+        private void db_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == Database.Fields.TNS)
+            {
+                Database db = (Database)sender;
+                //string intialValue = db.TNS;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5059209)